src/typescript-service.ts

```diff
@@ -113,6 +116,7 @@ export class TypeScriptService implements LanguageHandler {
 				// Tell the client that the server works in FULL text document sync mode
 				textDocumentSync: TextDocumentSyncKind.Full,
 				hoverProvider: true,
+				signatureHelpProvider: true,
```

Comment:
I'd say 
```
				signatureHelpProvider: {
					triggerCharacters: ['(', ',']
				},
```
This way it works in VS code when you hitting "(" or ","

BTW found a bug in our code while played with your PR, thanks! :smile: